gnome/gnome-voice/Portfile

```diff
@@ -23,12 +25,14 @@ checksums           rmd160  f15cae481f168b181858f9b86bb3fc2f2fb923ee \
                     sha256  1a58d20efc03c0eb9a7b540de93846b87d14eeb56908f74f61a53f1bd65d5dd7 \
                     size    105968
 
+perl5.major         5.34
+
 depends_build       port:autoconf \
                     port:automake \
                     port:gtk-doc \
                     port:intltool \
                     port:itstool \
-                    port:p5.28-xml-sax-expat \
+                    port:p${perl5.major}-xml-sax-expat \
```

Comment:
This one also doesn't seem to be doing anything to ensure the specified perl is used.

Reply:
Oh, I see we discussed this before. Even though it's only depends_build, it still needs to use the perl version that has `XML::SAX::Expat` installed.

Reply:
I tried adding (based on investigating the sources, I didn't try to actually configure or build anything, too many dependencies are needed, maybe I'll try later):
```tcl
perl5.major 5.28
configure.env-append INTLTOOL_PERL=${perl5.bin}
```
The fun part is that `rdeps` now show a mixture of perl 5.28 and 5.34 due to `intltool` depending on 5.34. I don't know what that actually means (whether the version needs to be compatible or not).

The port `intltool` has the comment:
```tcl
# intltool only uses perl internally and publishes no public modules
# so there is no need for multiple perl variants
set pbranch         5.34
```

@dbevans

Reply:
Oh OK, if the dependency is only there as a workaround to the INTLTOOL_PERL bug, then that should just be fixed properly and the dependency removed.

Reply:
Is this a widely known bug? How can one check the behaviour?

I tried removing the dependency, made sure that it wasn't installed, and destrooted with the `-vt` flag successfully. How do I diagnose whether it's still a bug or if the dependency can simply be removed?

```
checking for intltool >= 0.50.1... 0.51.0 found
checking for intltool-update... /opt/local/bin/intltool-update
checking for intltool-merge... /opt/local/bin/intltool-merge
checking for intltool-extract... /opt/local/bin/intltool-extract
checking for xgettext... (cached) /opt/local/bin/xgettext
checking for msgmerge... /opt/local/bin/msgmerge
checking for msgfmt... (cached) /opt/local/bin/msgfmt
checking for gmsgfmt... (cached) /opt/local/bin/msgfmt
checking for perl... /opt/local/bin/perl
checking for perl >= 5.8.1... 5.34.1
checking for XML::Parser... ok
checking that generated files are newer than configure... done
```

Reply:
@oleaamot